s3config.release.yaml.sample

```diff
@@ -89,7 +89,7 @@ S3_MOTR_CONFIG:                                     # Section for S3 Motr
                                                         # 1->4K, 2->8K, 3->16K, 4->32K, 5->64K, 6->128K, 7->256K, 8->512K, 9->1024K,
                                                         # 10->2048K, 11->4096K, 12->8192K, 13->16384K, 14->32768K, default is layout id 9 (1 MB)
    S3_MOTR_UNIT_SIZE: 1048576                         # Motr unit size w.r.t layout id for an IO operation
-   S3_MOTR_MAX_UNITS_PER_REQUEST: 32                  # Maximum units per read/write request to motr
+   S3_MOTR_MAX_UNITS_PER_REQUEST: 8                  # Maximum units per read/write request to motr
```

Comment:
You can add a comment here, saying that on hardware the value is set to 32, on VM/OVA its 8

Reply:
fixed